README.md

```diff
@@ -51,11 +57,37 @@ make install
 make install INSTALL_DIR=<your-path>
 ```
 
+### Enable gtctl autocompletion
```

Comment:
we may put this section under `Getting Started`, it seems not a proper place for `Installation`

Reply:
I think it's part of post-installation. Maybe the readme is too long, and moving it to `docs.greptime.com` will be better in the future.